src/api/java/de/fearnixx/jeak/reflect/RequestBody.java

```diff
@@ -10,12 +10,9 @@
  *
  * type(): REQUIRED Specify the class of the used variable.
  *
- * name(): REQUIRED Specify the name of the used variable.
- *
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.PARAMETER)
 public @interface RequestBody {
     Class<?> type();
```

Comment:
I would make this optional too. If the type is not specified, couldn't we just directly use the declared type of the method parameter?

Reply:
A good idea. It isn`t even required, so we could directly remove it.

Reply:
Removed it.